### ISSUE TITLE
Cannot click on buttons in cards - latest emulator

### ISSUE THREAD

**Issue by vishwacsena** [Microsoft/BotFramework-Emulator#837]
Create a bot that sends down a card (e.g. adaptive card)
Try and talk to the bot till it renders the adaptive card
Interact with the card (click on a button etc)

Expected: Card is interactive
Actual: Card buttons cannot be clicked since the message is selected and the inspector for it is highlighted and rendered. Subsequent clicks on the card or the message that holds the card does nothing. 

![image](https://user-images.githubusercontent.com/20918136/45257669-e2e8b780-b35e-11e8-8dab-54083c1e2090.png)

**Comment by vishwacsena** [comment]
To clarify input fields in adaptive cards work. Buttons however do not work. I've verified that the buttons in the card do work with the V3 emulator.

![image](https://user-images.githubusercontent.com/20918136/45257701-4246c780-b35f-11e8-8e00-3a3c5287be10.png)

**Comment by vishwacsena** [comment]
Adaptive cards with Action.Submit action seem to be the issue. 

Example card .. 
```json
{
    "type": "AdaptiveCard",
    "horizontalAlignment": "Center",
    "separator": true,
    "height": "stretch",
    "body": [
        {
            "type": "ColumnSet",
            "horizontalAlignment": "Center",
            "spacing": "large",
            "height": "stretch",
            "columns": [
                {
                    "type": "Column",
                    "spacing": "large",
                    "items": [
                        {
                            "type": "TextBlock",
                            "size": "extraLarge",
                            "weight": "bolder",
                            "text": "Contoso Cafe"
                        },
                        {
                            "type": "TextBlock",
                            "size": "Medium",
                            "text": "Hello, I'm the Cafe bot! How can I be of help today?",
                            "wrap": true
                        }
                    ]
                },
                {
                    "type": "Column",
                    "spacing": "small",
                    "items": [
                        {
                            "type": "Image",
                            "horizontalAlignment": "center",
                            "url": "http://contosocafeontheweb.azurewebsites.net/assets/contoso_logo_black.png",
                            "size": "medium"
                        }
                    ],
                    "width": "auto"
                }
            ]
        }
    ],
    "actions": [
        {
            "type": "Action.Submit",
            "title": "Book table",
            "data": {
                "intent": "Book_Table"
            }
        },
        {
            "type": "Action.Submit",
            "title": "What can you do?",
            "data": {
                "intent": "What_can_you_do"
            }
        }
    ],
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "version": "1.0"
}
```

**Comment by compulim** [comment]
Cannot repro on my box using your Adaptive Card.

![image](https://user-images.githubusercontent.com/1622400/45330603-09ebe880-b51b-11e8-8dc4-1c48cb5d014b.png)

**Comment by vishwacsena** [comment]
@compulim - I continue to repro this in the latest emulator build 
![image](https://user-images.githubusercontent.com/20918136/45401633-637d1180-b606-11e8-9205-002c88b00046.png)

**Comment by compulim** [comment]
Investigating now.

**Comment by compulim** [comment]
I tried to compile using the source code on 39111, it works.

But when I use the binary, it doesn't.

Investigating.

**Comment by compulim** [comment]
One thing looks weird to me.

![image](https://user-images.githubusercontent.com/1622400/45452537-185d1000-b693-11e8-8bef-fdbbaaebb1c5.png)

When `ReactDOM.findDOMNode(this.historyRef)` is called, `react-dom` throw an exception saying `historyRef` is unmounted. This only appears on installed version of Emulator, not on development build.

I double-checked `this.historyRef`, it was not unmounted, and we did not miss any `saveRef` calls. It feels so weird to me: if a component is unmounted, we should receive `saveRef` call, but we didn't. And what's more, we should not have unmounted `historyRef` anyway.

Since the `historyRef.focus` is optional, I am try-catching it on Web Chat side for now.

**Comment by loomlike** [comment]
I'm having the same issue w/ OAuthCard.
Card is rendered at the emulator properly, but cannot click the signin button in the card from the emulator.
Just the message gets highlighted.

**Comment by loomlike** [comment]
nvm, I had to add AppID and PW to make OAuthCard work... so, now click works.

**Comment by compulim** [comment]
I would not close it but down it to P1 or P2.

The reason is, this is a quirk and we are not actually fixing it. It only repro on setup binaries, but not on development branch, that could means a broader problem.

I would wait until someone verified it on next nightly, and then downgrading it for post-Ignite.

p.s. I am thinking, if the version of React using by Web Chat and Emulator are different in setup binaries (not hoisted?) and development branch (hoisted)

**Comment by justinwilaby** [comment]
@vishwacsena  - PR #901 looks like it fixed this problem. Can your confirm and close?

**Comment by tonyanziano** [comment]
@compulim This is fixed in v3 correct?

**Comment by cwhitten** [comment]
Addressed in #901
